cypress/support/step-definitions/common-steps.js

```diff
@@ -146,6 +153,33 @@ Then('I hover mouse onto {string} icon into iFrame', (name) => {
   getElementNoIframe(name).trigger('mouseover');
 });
 
+Then('I hover mouse onto {string} {string} icon for validations component into iFrame', (position, name) => {
```

Comment:
probably we should move this step to Validation-steps ?

Reply:
This step is used in two different components, that's the reason why I put it here --> checkbox and validations component.